tests/cpp/test_multidevice_overlap.cpp

```diff
@@ -18,6 +18,28 @@
 
 namespace nvfuser {
 
+namespace {
+
+std::vector<c10::cuda::CUDAStream> CreateStreams(
```

Comment:
```suggestion
std::vector<c10::cuda::CUDAStream> createStreams(
```
for consistency with the rest of the code base.